functions/src/Controller/Committee.ts

```diff
@@ -1,4 +1,4 @@
-import { addCommittee, getCommittees, editCommittee, deleteCommittee } from '../Services/Committee';
+import { addCommittee, getCommittees, editCommittee, deleteCommittee, changeLevel } from '../Services/Committee';
```

Comment:
maybe rename changeLevel to changeCommitteeLevel? I think it would be more informative as to what the function is doing. (This is very minor, you can keep it the same if you want)

Reply:
Since it's in the Committee file and it's imported from /Services/Commitee I think it's self-explanatory for anyone using it.

Reply:
I agree with Bautista I think it should be changeCommitteLevel not changeLevel because that could get confused with something else